src/components/Header/Header.module.css

```diff
@@ -1,74 +1,21 @@
-.Header {
-    background-color: #103F5F;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
-    align-items: center;
-    color: white;
-    padding: 0.5rem 1rem;
-    z-index: 9999;
-    font-weight: bold;
-    width: 100%;
-    position: sticky;
-    top: 0;
-    box-shadow: 0px 1px 5px white;
-}
-
 .left {
-    object-fit: contain;
-    width: 170px;
+  object-fit: contain;
+  width: 100px;
 }
-
 .right {
-    display: flex;
-    justify-content: center;
-    align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
 }
 
 .header_links {
-    padding: 0.3rem 0.5rem;
-    margin: 0 0.1rem;
-    border-radius: 15px;
-    cursor: pointer;
-    color: white;
-    text-decoration: none;
+  padding: 0.3rem 0.5rem;
+  margin: 0 0.1rem;
+  border-radius: 15px;
+  cursor: pointer;
 }
 
 .header_links:hover {
-    background-color: white;
-    color:#103F5F;
-    text-decoration: none;
-}
-
-.header_active_links {
-    box-shadow: inset 1px 1px 3px grey;
-    background-color: lightblue;
-    color:#103F5F;
-    text-decoration: none;
+  background-color: white;
+  color: #103f5f;
 }
-
-.quick_box {
```

Comment:
Why you have removed these styles for the Quick Box? Due to this, Quick Links are now hidden beside the Header.

Reply:
@nehaparmar99 I believe you have not pulled the recent changes to the master branch